Source/init.cpp

```diff
@@ -160,6 +161,9 @@ void init_archives()
 	paths.push_back("/usr/local/share/diasurgical/devilutionx/");
 #endif
 
+	// An example of logging a vector of strings.
+	Log("Init: paths = {}", paths);

```

Comment:
This is already logged below with `SDL_LogVerbose`; it shouldn't be logged twice.

Reply:
Yes this was only an example of logging a std::vector.